mappings/net/minecraft/registry/Registries.mapping

```diff
@@ -23,7 +23,7 @@ CLASS net/minecraft/unmapped/C_nusqeapl net/minecraft/registry/Registries
 	FIELD f_hmxbehsm CUSTOM_STAT Lnet/minecraft/unmapped/C_tqxyjqsk;
 	FIELD f_iaquflcq MEMORY_MODULE_TYPE Lnet/minecraft/unmapped/C_zogerkic;
 	FIELD f_idnjesqk ROOT_REGISTRY_KEY Lnet/minecraft/unmapped/C_ncpywfca;
-	FIELD f_ipaoqeaf FEATURE_WORLDGEN Lnet/minecraft/unmapped/C_tqxyjqsk;
+	FIELD f_ipaoqeaf FEATURE Lnet/minecraft/unmapped/C_tqxyjqsk;
```

Comment:
In this case it's clearer to use `WORLDGEN_FEATURE` rather than just `FEATURE` imo

Reply:
true, it's possible it could cause confusion with people thinking they have to register any sort of feature they add to the game